nixos/modules/services/x11/window-managers/oroborus.nix

```diff
@@ -0,0 +1,25 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.oroborus;
+in
+{
+  ###### interface
+  options = {
+    services.xserver.windowManager.oroborus.enable = mkEnableOption "oroborus" { };
```

Comment:
`mkEnableOption "foo"` evaluates to a set, which means you are trying to apply a non-function value here. What you want is simply `mkEnableOption "oroborus";`.